scripts/create_release_tag.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/bash -ex
+#!/bin/bash
```

Comment:
We probably want the `-e` to stay so that the script fails on any intermediate errors.